samples/archived/database-get-default-leader.js

```diff
@@ -1,69 +0,0 @@
-/**
```

Comment:
Just to put it on record - Can you answer why this file is being removed from `archived` folder?